src/containers/AlbumsView.jsx

```diff
@@ -15,13 +15,20 @@ export class AlbumsView extends Component {
   constructor (props) {
     super(props)
     this.state = {isFetching: true, error: false}
+
+    this.handleErrorFromChildren = this.handleErrorFromChildren.bind(this)
+
     props.fetchAlbums()
       .then(() => { this.setState({isFetching: false, error: false}) })
       .catch(albumsError => {
         this.setState({isFetching: false, error: albumsError})
       })
   }
 
+  handleErrorFromChildren (error) {
```

Comment:
I am not a big fan of the naming here. I think that we should be able to handle on error whatever its source is.

Si in my opinion, this method should be called `onError` and pass to children component via an attribute with the same name. It feels more idiomatic. What do you think ? What's @goldoraf opinion ?